assets/js/main.js

```diff
@@ -474,6 +474,7 @@ class Carousel {
             var left = 50 + 100.0 / itemsRow * idxDiff;
             // Allows image overlap with each other.
             var width = carousel.overlapFactor / itemsRow;
+
```

Comment:
This commit contains nothing lolololol

Reply:
I think this commit was created because I did the rebase, and then fixed some conflict (where I made that carousel.itemRows mistake), then push this...